ee/modules/380-metallb/templates/controller/deployment.yaml

```diff
@@ -93,6 +99,26 @@ spec:
                       resource: deployments
                       subresource: prometheus-metrics
                       name: metallb
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /livez
+              port: https-metrics
+              scheme: HTTPS
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /livez
+              port: https-metrics
+              scheme: HTTPS
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
```

Comment:
livenessProbe == readinessProbe — this may be a problem.

Reply:
i removed readiness probe because metallb do not have special logic for readiness.

Reply:
I removed all probes at all, because metallb does not have any reasonable probe endpoints.